drltrace/drltrace.config

```diff
@@ -1,3 +1,38 @@
+; ***************************************************************************
```

Comment:
IMHO # be used for a comment instead of ; as # is more commonly used that way in config files and scripts.